c/CHANGELOG.rst

```diff
@@ -1,3 +1,11 @@
+--------------------
+[2.0.2] - 2022-01-XX
```

Comment:
This is adding a new feature, so I think it's a minor version bump rather than a patch bump?